crypto/keys/der/proto/register.go

```diff
@@ -17,21 +17,11 @@
 package proto
 
 import (
-	"context"
-	"crypto"
-	"fmt"
-
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/crypto/keys/der"
 	"github.com/google/trillian/crypto/keyspb"
-	"google.golang.org/protobuf/proto"
 )
 
 func init() {
-	keys.RegisterHandler(&keyspb.PrivateKey{}, func(ctx context.Context, pb proto.Message) (crypto.Signer, error) {
-		if pb, ok := pb.(*keyspb.PrivateKey); ok {
-			return der.FromProto(pb)
-		}
-		return nil, fmt.Errorf("der: got %T, want *keyspb.PrivateKey", pb)
-	})
+	keys.RegisterHandler(&keyspb.PrivateKey{}, der.FromProto)
```

Comment:
Is the unused ctx passed in FromProto forced by RegisterHandler here?

Reply:
That's right, it has to match that function signature.